IOSAccessAssessment/Views/SetupView.swift

```diff
@@ -8,17 +8,40 @@
 import SwiftUI
 
 struct SetupView: View {
+    
+    private enum SetupViewConstants {
```

Comment:
Again, I do believe it would be good to centralize the constants, even if they are hard-tied to some views at this moment. 
Maybe we can change the 'Model' folder (which tbh isn't serving as a Model folder anymore) to 'Constants', and house the main constants over there.